Update render mode selection

Jmlundeen opened the pull request:
* changes render mode selection to a drop-down
* modes include MTGO, Image, Forced M15, Forced Retro
* debug card test panel updated with additional modes as well
PreferencesDialog.getRenderMode() uses the enum ordinal to reduce the amount of refactoring

![image](https://github.com/user-attachments/assets/44c75537-3955-4fcd-b255-ddb73f60fb8a)
![image](https://github.com/user-attachments/assets/b79d044e-c5df-49be-a523-c5f2e473bca7)
![image](https://github.com/user-attachments/assets/9fe66bcc-533c-447a-aa79-d8264451f10e)

xenohedron commented:
Anything in particular you want a reviewer to look at? I definitely endorse the direction

Jmlundeen commented:
Wording of setting names and reminder text mostly. Also, I figured @JayDi85 would want to look at it first before merging.

xenohedron commented:
For the mouseover descriptions, maybe make it more clear that forced m15/retro are a variation on mtgo style?

Can you include a screenshot of the new debug panel for reference?

Jmlundeen commented:
Finally got back to this, updated the wording on the tooltip and added it to the debug panel as well.
![java_oqmIdMRRq6](https://github.com/user-attachments/assets/6835dbf0-dbcc-4887-b4be-0d945be5b803)

JayDi85 commented on Mage.Client/src/main/java/mage/client/dialog/PreferencesDialog.java:
```diff
@@ -3815,15 +3838,7 @@ public static String getCachedValue(String key, String def) {
-
-    public static boolean getRenderRetroFrames() {
-        return (getCachedValue(PreferencesDialog.KEY_CARD_RENDERING_ENABLE_RETRO_FRAMES, "true").endsWith("true"));
+        return CardRenderMode.fromString(getCachedValue(PreferencesDialog.KEY_CARD_RENDERING_IMAGE_MODE, CardRenderMode.MTGO.toString())).ordinal();
```
Add id or like that to enum and use it here, do not use ordinal.

JayDi85 commented on Mage.Client/src/main/java/mage/client/util/CardRenderMode.java:
```diff
@@ -0,0 +1,44 @@
+                return mode;
+            }
+        }
+        return IMAGE;
```
Make mtgo as default — it’s better look without images, e.g. on first run by a new user

JayDi85 reviewed (commented):
Few small fixes require, but all other is fine
